content/en/blog/_posts/2019-03-12-kubeedge-k8s-based-edge-intro.md

```diff
@@ -0,0 +1,114 @@
+
+Open source edge computing is going through its most dynamic phase of development in the industry. So many open source platforms, so many consolidations and so many initiatives for standardization! This shows the strong drive to build better platforms to bring cloud computing to edges to meet ever increasing demand. KubeEdge, which was announced last year, now, brings great news for cloud native computing! It provides a complete edge computing solution based on kubernetes with separate cloud and edge core modules. Currently it has both cloud and edge modules open sourced. It can work in offline mode, light weight, containerized, and support heterogeneous hardware at edge. This makes it the most compelling edge computing platforms in the world currently, based on Kubernetes!
+
+
+## Kube(rnetes)Edge! - Opening up a new kubernetes based ecosystem for Edge Computing
+
+The key goal for KubeEdge is extending Kuberentes ecosystem from cloud to edge. From the time it was announced to public (Shanghai KubeCon, Nov, 2018), the architecture direction for KubeEdge was aligned to Kubernetes, as its name!
+
+It started with its v0.1 providing the basic edge computing features. Now, with its latest release v0.2, it brings the cloud components to connect and complete the loop. With consistent and scalable kubernetes based interfaces, KubeEdge enables the orchestration and management of edge clusters similar to how Kubernetes manages in the cloud. This opens up seamless possibilities of bringing cloud computing capabilities to the edge, quick and efficient.
+
+<p align="center">
+<img src="../../../../static/images/blog/2019-03-12-kubeedge-k8s-based-edge-intro/kubeedge-logo.png">
+</p>
+
+**KubeEdge Links:**
+
+  - [Website](https://kubeedge.io)
+
+  - [SourceCode](https://github.com/kubeedge/kubeedge)
+
+  - [Documentation](https://docs.kubeedge.io)
+
+
+Based on its roadmap and architecture, it tries to support all edge nodes, applications, devices and even the cluster management consistent with kuberenetes interface. This will help the edge cloud enact exactly like a cloud cluster. This can save a lot of time and cost on the edge cloud development deployment based on KubeEdge.
+
+KubeEdge provides a containerized edge computing platform, which is inherently scalable. As it's modular and optimized, it is light weight (66MB foot print and ~30MB running memory) and could be deployed on low resource devices. Similarly, the edge node can be of different hardware architecture and with different hardware configurations. For the device connectivity it can support multiple protocols (getting extended the list) and it uses a standard MQTT based communication. This helps in scaling the edge clusters with new nodes and devices efficiently. 
+
+
+**<p align="center">You heard it right!</p>**
+**<p align="center">KubeEdge Cloud Core modules are open sourced!</p>**
+
```

Comment:
what's thoes two line means?

Reply:
Highlighting in between the blog these texts to call out that point. Aligned to center.